### ISSUE TITLE
Routing

### ISSUE THREAD

**Issue by Robinlovelace** [creds2/od-data#2]
Top 100k lines? I suggest starting with top 10k, look at results, and take it from there.

**Comment by mem48** [comment]
OTP is building overnight, now with new config files.

**Comment by Robinlovelace** [comment]
Fantastic. How did it go?

**Comment by mem48** [comment]
@Robinlovelace I've uploaded as a release a first attempt at the routing. It was 84% successful at getting the 100,000 routes and took 19 hours. Slower than I hoped but still beets using 2,500 per day with google / ORS. The failed routes seem to fall into 2 categories.

1. OTP timed out before finding the route (it had a 5-second cap)

2. The old problem of centroids being too far from the road.

I've set it up to run again with a 20-second timeout to see if that fixes the bulk of the long routes. For as for us long routes are especially interesting. 

As a point of interest on UK wide graph OTP seems to average 1.4 routes per second, so that would work out as 8 days per 1 million routes. But I suspect that may be an overestimate for the commuting data as this is based on a biased sample of longer routes.
